### ISSUE TITLE
Hide System Field Updates in Record Page Timelines

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/twenty#6761]
# Current Behavior

Some system field updates are displayed in record page timelines. However,  as they lack a FieldDisplay, no value appears near the field title.

![CleanShot 2024-08-28 at 11 48 36](https://github.com/user-attachments/assets/c73bd188-f419-4713-9afb-59869939ac22)

# Desired Behavior

We shouldn't display system fields in the timeline.

**Comment by Faisal-imtiyaz123** [comment]
@Bonapara I would like to work here.

**Comment by Bonapara** [comment]
Sure @Faisal-imtiyaz123, thanks for contributing!

**Comment by charlesBochet** [comment]
@Bonapara @FelixMalfait should we rather add a isAuditable on field level?

I don't like the isSystem one :p

**Comment by FelixMalfait** [comment]
@charlesBochet At the field-level it's a bit more subtle than `isAuditable` because we still want these fields to be audit-logged but we don't want the timelineActivity to display include them, so I'd advocate for a different name (`isDisplayedInTimelineActivity` or something like that).
But I don't see a case where a field would be displayed in the page, and wouldn't be displayed in the timeline.
So in my opinion, the field we should use here is the same we will introduce to replace most use-cases for `isSystem` (`isDisplayed` or something like that).
Bottom line is I think we should use `isSystem`

**Comment by charlesBochet** [comment]
Okay I understand the reasoning! Let's go for isSystem as long as we don't have a better naming

**Comment by FelixMalfait** [comment]
Changing to GoodFirstIssue as it shouldn't be too difficult. 
We should filter out fields where isSystem = true in the diff.
Think about the edge case where just 1 field was changed (e.g. position was changed by dragging the item on the list view), then we don't want to create any timelineActivity.

Do it on the backend rather than on the frontend. I would probably do it in `EntityEventsToDbListener`

**Comment by Bonapara** [comment]
/oss.gg 150

**Comment by Khaan25** [comment]
Hey @Bonapara, May I ask what are the steps to reproduce it? I've tried to find the Position to change but could find any field.

**Comment by Bonapara** [comment]
Sure: 

https://github.com/user-attachments/assets/957f71aa-8d6e-4b53-af9a-354684c99214

**Comment by Khaan25** [comment]
@Bonapara, I've fixed other issues I've had. I would love to work on this issue as well.

**Comment by Bonapara** [comment]
@razamzai are you still on it?

**Comment by Khaan25** [comment]
Hey, could you assign me :)

My other PRs is under review. I can quickly wrap this up as well
